feature-utils/poly-analysis/src/analysis/report/inactive-cards-summary.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+
+/**
+ * I am a "fake" analysis that shows issues when running analyses.
+ * I do not have an analyse method as I get the data directly in the constructor.
+ *
+ * I provide the same API as an analysis that should be included in a report.
+ *
+ * @class
+ */
+export default class InactiveCardsSummary {
+    constructor(analysesResults) {
+        const inactiveAnalysesResults = analysesResults.filter(
+            ({ analysis, status }) => status.isError || !analysis.active
+        );
+
+        this._inactiveAnalysesData = inactiveAnalysesResults.map(
+            ({ analysis, status }) => {
+                return {
+                    analysis: analysis.id,
+                    activationStatus: analysis.active ? "ACTIVE" : "INACTIVE",
+                    executionStatus: status.name,
+                    message: status.message,
+                };
+            }
+        );
+        this.active = this._inactiveAnalysesData.length > 0;
```

Comment:
This function is doing two different things, which is not a good separation of concerns. Since it's a constructor, not a big deal. Not in scope, but what's in scope is that `this._inactiveAnalysesData` should have exactly the same length as `inactiveAnalysesResults` since it's a map.